include/server/api/base_parameters_grammar.hpp

```diff
@@ -149,10 +159,13 @@ struct BaseParametersGrammar : boost::spirit::qi::grammar<Iterator, Signature>
             qi::lit("bearings=") >
             (-(qi::short_ > ',' > qi::short_))[ph::bind(add_bearing, qi::_r1, qi::_1)] % ';';
 
+        sides_rule = qi::lit("sides=") >
```

Comment:
I tryed to change by this code :
```
        side_rule = qi::char_("a-zA-Z")[qi::_val = engine::Side{engine::BOTH}];
        struct StartSideValues: qi::symbols<char, engine::Side>
        {
            StartSideValues() {add("b", engine::Side{engine::BOTH})("d", engine::Side{engine::DEFAULT})("o", engine::Side{engine::OPPOSITE})("", engine::Side{engine::OPPOSITE}); }
        };
        StartSideValues values;
        sides_rule = qi::lit("sides=") >
                        (-(values | side_rule) %
                         ';')[ph::bind(&engine::api::BaseParameters::sides, qi::_r1) = qi::_1];
```
But it provokes segfault.
I am trying to analyse with gdb but it's a really complicated backtrace.